src/app/acesso/cadastro/cadastro-prestador/cadastro-prestador.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroupDirective, NgForm, FormBuilder,FormGroup, Validators} from '../../../../../node_modules/@angular/forms';
+import { Router } from '@angular/router';
+import { LocalStorageService } from '../../../servicos/local-storage.service';
+import { UsuarioService } from '../../../servicos/usuario.service';
+import { PrestadorService } from '../../../servicos/prestador.service';
+import { Prestador } from '../../../interfaces/prestador';
+import { TipoPessoa } from '../../../enum/tipo-pessoa.enum';
+import {MyErrorStateMatcher} from '../../../classes/my-error-state-matcher';
+import { Estado, Cidade, ConsultaEstadosService } from 'src/app/servicos/consulta-estados.service';
+import { LoginService } from 'src/app/servicos/login.service';
```

Comment:
Codacy found an issue: [`src/app/servicos/login.service` import should occur before import of `../../../../../node_modules/@angular/forms`](https://app.codacy.com/manual/gaugsouza/petshow-frontend/pullRequest?prid=6332072)